src/main/java/com/google/sps/servlets/MemesServlet.java

```diff
@@ -87,5 +87,40 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
       String imgTag = String.format("<img src=\"%s\" width = \"300\"/>", blob.getMediaLink());
       response.getWriter().println(imgTag);
     }
+  }
+
+/** This funtions  randomly  get a string  captions from an array of string. When the function is called
+ * it randomly return memes caption from an array
+ * input: null
+ * return: strings..
+ * 
+ */
+  
+
+  public static String getCaption(){
+//     initilaise an array  of captions
+        final String[] captionList = {"Fred", 
+        
+        "When the professor is passionate about teaching and you genuinely understan and enjoy the class.", 
+        " When ur best friend calls u and have some gossip to tell u.", 
+        "When you go off on him and later on realize you were actually wrong",
+        "When you tell everyone about your summer body goals, but you didn’t tell them which summer",
+
+        "  When you’ve been eating healthy for the past 15 minutes and STILL see no progress.",
+
+        
+    
+    };
+    //   import the random module
+        Random random = new Random();
+        // get random  index of array element
+        int index = random.nextInt(captionList.length);
+       // System.out.println(proper_noun[index]);
```

Comment:
Next time, you can delete commented out code just to keep the main repo cleaner :)